src/app/helptext/storage/volumes/datasets/dataset-unlock.ts

```diff
@@ -6,7 +6,8 @@ export const helptextUnlock = {
   unlock_key_file_placeholder: T('Unlock with Key file'),
   unlock_key_file_tooltip: T('Use an exported encryption key file to unlock datasets.'),
   unlock_children_placeholder: T('Unlock Child Encrypted Roots'),
-  unlock_children_tooltip: T('Also unlock any encrypted dataset stored within this dataset.'),
+  unlock_children_tooltip: T('Also unlock any encrypted dataset stored within this dataset. \
```

Comment:
Let's change first line to "Also unlock any separate encryption roots that are children of this dataset".